decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -129,6 +129,11 @@ def patch_test_file
                   "# config.action_controller.raise_on_missing_callback_actions = false"
       end
 
+      def patch_development_file
+        gsub_file "config/environments/development.rb", /config\.action_view\.annotate_rendered_view_with_filenames = true$/,
+                  "# config.action_view.annotate_rendered_view_with_filenames = true"
+      end
+
```

Comment:
Lets make the method name more descriptive, as there are other methods changing development too. 
Also lets add more details in the actual change

```suggestion
      def disable_annotate_rendered_view_on_development
        gsub_file "config/environments/development.rb", /config\.action_view\.annotate_rendered_view_with_filenames = true$/,
                  "# Using annotate rendered view breaks rails-ujs functionality\r
  # @see https://github.com/decidim/decidim/issues/14912\r
  # config.action_view.annotate_rendered_view_with_filenames = true"
      end

```

Reply:
Fixed via 1a0941cf03